README.md

```diff
@@ -1,2 +1,42 @@
-# cloud-platform-prometheus
+# Cloud-Platform Prometheus
+==================
+
+This repository will allow you to create a monitoring namespace in a MoJ Cloud Platform cluster. It will also contain the neseccary values to perform an installation of Prometheus-Operator and Kube-Prometheus. 
+
+  - [Pre-reqs](#prereq)
+  - [Creating a monitoring namespace](#installation)
+  - [Installing Prometheus-Operator](#output-example)
+  - [Installng Kube-Prometheus](#usage)
+  - [Exposing the port](#features-and-advantages-of-this-project)  
+  - [How to tear it all down](#am-i-missing-some-essential-feature)
+  - [Contributing](#contributing)
+
+```
+TL;DR
+# Copy dir to your namespace in the cloud-platform-environments repository.
+
+# Add CoreOS Helm repo
+$ helm repo add coreos https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
+
+# Install prometheus-operator
+$ helm install coreos/prometheus-operator --name prometheus-operator --namespace monitoring -f ./helm/prometheus-operator/values.yaml
+
+# Install kube-prometheus
```

Comment:
As discussed, I'm not sure that we need to do this as we can set up everything via the operator. Would be good to see where you end up with it.

Reply:
Yes, will have a chat to Mitch about this.